src/components/shared/accountVisualWithAddress/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import styles from './accountVisualWithAddress.css';
+import Icon from '../../toolbox/icon';
+import transactionTypeIcons from '../../../constants/transactionTypeIcons';
+import transactionTypes, { transactionNames } from '../../../constants/transactionTypes';
+import AccountVisual from '../../toolbox/accountVisual';
+import regex from '../../../utils/regex';
+
+class AccountVisualWithAddress extends React.Component {
+  getTransformedAddress(address) {
+    const { isMediumViewPort, bookmarks, showBookmarkedAddress } = this.props;
+
+    if (showBookmarkedAddress) {
+      const bookmarkedAddress = bookmarks.LSK.find(element => element.address === address);
```

Comment:
IMO it's more future prone to use active token from store `settings.token.active` here:
```suggestion
      const bookmarkedAddress = bookmarks[this.props.token.active].find(element => element.address === address);
```

Reply:
✅